Packages/Core/Editor/Generator/Templates/PairTemplateFactory.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using UnityEngine;
+
+namespace UnityAtoms.Editor
+{
+    public static class PairTemplateFactory
```

Comment:
Why does this one look so different from the others?

Reply:
Instead of writing a GetPairTemplate method for every pair, I instead wrote a GetTemplate method that modifies an existing GetTemplate method to turn that template into a pair. That may be pretty confusing, but just look how it is used in the EventTemplateFactory. It generates both an AtomEvent and an AtomEventPair from the same GetTemplate method, but one is parsed through the PairTemplateFactory.